### DISCUSSION TITLE
Octoprint's Firmware Updater plugin

### DISCUSSION THREAD

**Discussion by jjfs127** [Jyers/Marlin#67] [Q&A] [answered]
Just a general question, can we upgrade the V2's firmware via Octoprint's Firmware Updater plugin, instead of constantly using the SD card? I believe it uses avrdude.
Great work btw!

**Comment by Schnabulation** [comment]
...I wondered this as well. With me trying out every new commit Jyers publishes it had me running back to grab the SD card in the printer a lot lately...

**Comment by Jyers** [comment]
I have given this a few tries, as I end up flashing a lot of firmware during testing. As of yet I have never managed to get the board to reset with any of the available pin settings. If anyone has any info on how to get it working properly I would be immensely grateful

**Reply by t3chguy** [reply]
same experience here

**Comment by Sennevds** [comment, marked as answer]
There's an issue about the v2 in the repo of the plugin https://github.com/OctoPrint/OctoPrint-FirmwareUpdater/issues/190
